Week02/lesson04/homework/Layout Challenge/Two Column/styles.css

```diff
@@ -0,0 +1,15 @@
+* {
+  -webkit-box-sizing: border-box;
+}
+
+  .top {
+  background-color: #ff6666;
+  border: 2px solid red;
+  margin: 2px;
+  height: 150px;
+}
+
+.middle {
+  display: inline-block;
+  width: 50%;
```

Comment:
So I'm guessing here you were wondering WTF was going on. You did all the right things, you made the two divs `inline-block`s, you set the width to `50%`, now WHY are they not sitting side-by-side right? This is due to a VERY weird phenomenon that we'll be covering tomorrow, so don't worry, this is a wall you were meant to run into.

Honestly, you did everything right. This is just one of those weird little things kind of like how floats are pulled out of the stacking order that can really be explained better in person, so hang tight.